spec/controllers/api_controller_spec.rb

```diff
@@ -178,15 +178,16 @@
   describe 'GET #delegates' do
     it 'includes emails and regions' do
       senior_delegate = FactoryBot.create :senior_delegate_role
-      delegate = FactoryBot.create :delegate, location: "SF bay area, USA", region_id: senior_delegate.group.id
+      delegate = FactoryBot.create :delegate_role, group_id: senior_delegate.group.id
+      delegate.metadata.update!(location: "SF bay area, USA")
```

Comment:
I'm not super happy about this hard-wired update call in tests.
Would it be a lot of trouble to pass through the `location` to the `metadata` in the FactoryBot macro?